libs/ui/src/bmd_paper_ballot.tsx

```diff
@@ -51,11 +51,15 @@ const Ballot = styled.div`
   }
 `;
 
-const Header = styled.div`
+interface StyledHeaderProps {
+  largeTopMargin?: boolean;
+}
+const Header = styled.div<StyledHeaderProps>`
   display: flex;
   flex-direction: row;
   align-items: center;
   border-bottom: 0.2em solid #000;
+  margin-top: ${(p) => (p.largeTopMargin ? '1.75in' : undefined)};
```

Comment:
Not sure if this is the right way to fallback, but I explicitly don't want to overwrite margin with `0` in the case where `largeTopMargin` is falsy.

Reply:
If it works, then seems fine to me!